### DISCUSSION TITLE
Table block

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1928] [Ideas]
This would be a useful addition. Currently, the only way I see to add a table is via a Custom Code block.

**Comment by meetmandeep** [comment]
Can you share a use case? Tables are hardly used in modern web design in lieu of Card Layouts and other responsive design patterns. We are specifically steering away from use of tables.

**Comment by eswanzey** [comment]
What better way to present scientific or structured data when you have to build it up yourself as plain old content.

It could have a block editor that mimics a spreadsheet, so editors are shielded from code.

![2021-11-29_16-14-29](https://user-images.githubusercontent.com/8103212/143944510-171db614-0ff9-44f4-9c39-d79f9f3fc174.jpg)
